package.json

```diff
@@ -2,19 +2,25 @@
   "name": "stockflux",
   "version": "13.0.0",
   "devDependencies": {
+    "concurrently": "^4.1.2",
     "lerna": "^3.16.4"
   },
   "scripts": {
     "bootstrap": "lerna bootstrap --concurrency 1",
     "build": "lerna run build",
-    "start-chart": "cd packages/stockflux-chart && npm run start",
-    "start-container": "cd packages/stockflux-container && npm run start",
-    "start-launcher": "cd packages/stockflux-launcher && npm run start",
-    "start-news": "cd packages/stockflux-news && npm run start",
-    "start-watchlist": "cd packages/stockflux-watchlist && npm run start",
-    "launch-chart": "cd packages/stockflux-chart && npm run launch",
-    "launch-launcher": "cd packages/stockflux-launcher && npm run launch",
-    "launch-news": "cd packages/stockflux-news && npm run launch",
-    "launch-watchlist": "cd packages/stockflux-watchlist && npm run launch"
+    "chart:start": "cd packages/stockflux-chart && npm run start",
+    "container:start": "cd packages/stockflux-container && npm run start",
+    "launcher:start": "cd packages/stockflux-launcher && npm run start",
+    "news:start": "cd packages/stockflux-news && npm run start",
+    "watchlist:start": "cd packages/stockflux-watchlist && npm run start",
+    "start:all": "concurrently \"npm run chart:start\" \"npm run container:start\" \"npm run launcher:start\" \"npm run news:start\" \"npm run watchlist:start\"",
```

Comment:
Can you rename this script to `all:start`, please? Just to keep it consistent with the others.

Reply:
Yeah, good point, thanks.